EventView-1/src/main/java/com/eventview/exceptions/UserExceptionController.java

```diff
@@ -0,0 +1,28 @@
+package com.eventview.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.sql.Timestamp;
+
+@ControllerAdvice
+public class UserExceptionController {
+    @ExceptionHandler(value = UserNotFoundException.class)
+    public ResponseEntity<ErrorResponse> userNotFoundException(UserNotFoundException exception) {
```

Comment:
same as above what is diff this and EventExceptionController , u shd combine all in same without duplicates